src/api/rootme_api.py

```diff
@@ -0,0 +1,41 @@
+from os import getenv
+
+from api.rate_limiter import RateLimiter
+import requests
+import time
+
+class RootMeApi() :
+    """
+    Class that provides information fetched from the Root-me.org API
+     -> needs a rate_limiter object to send all the requests at a maximum rate of 25 req/sec
+    """
+
+    def __init__(self,rate_limiter : RateLimiter) :
+        if getenv("API_KEY_ROOTME") is not None :
+            self.API_KEY = getenv("API_KEY_ROOTME")
+        else :
+            raise Exception("API_KEY_ROOTME is not set.")
+        self.url = "https://api.www.root-me.org/"
```

Comment:
maybe get this one from env? or at least global constant?

Reply:
Yes indeed :+1: